types/xdr.d.ts

```diff
@@ -2056,6 +2227,124 @@ export namespace xdr {
     static validateXDR(input: string, format: 'hex' | 'base64'): boolean;
   }
 
+  class LiquidityPoolConstantProductParameters {
+    constructor(attributes: { asseta: Asset; assetB: Asset; fee: number });
+
+    asseta(value?: Asset): Asset;
+
+    assetB(value?: Asset): Asset;

```

Comment:
All parameters with the pattern `*assetA` were generated with the camelCase wrong:
* asseta should be assetA
* maxAmounta should be maxAmountA
* minAmounta should be minAmountA

Reply:
Seems like the problem is the ruby `pluralize` function in [stellar/xdrgen](https://github.com/stellar/xdrgen/blob/9dacf24a73b3e73105e9272ccf333704dcc82333/lib/xdrgen/generators/javascript.rb#L181-L183).

Everything ending with `ta` (or `tA`) is not pluralized, so the assertion below will always be true:
```rb
plural = named.name.downcase.pluralize == named.name.downcase  # true, beacuse `asseta == asseta`
```

Reply:
Adding a fix in https://github.com/stellar/xdrgen/pull/62

Reply:
Re-generated the code and fixed those typos in https://github.com/stellar/js-stellar-base/pull/454/commits/714c66b12c4eae4f40dd874f650f75cd248206b9.